src/pages/signing_official_console/SigningOfficialTable.jsx

```diff
@@ -378,10 +378,10 @@ export default function SigningOfficialTable(props) {
               fontWeight: 600,
               fontSize: '2.8rem' }}
             >
-              My Institution&apos;s Library Cards
+              Pre-authorize my Institution&#39;s Researchers with Library Cards
```

Comment:
I'm wondering if this should just be Authorize since revoking a library card also suspends data access.
```suggestion
              Authorize my Institution&#39;s Researchers with Library Cards
```